.github/workflows/release-update-github-page.yml

```diff
@@ -2,8 +2,8 @@
 # GitHub Action to Publish Release Notes GitHub Page.
```

Comment:
I believe this is the same, not in synch with name.

Reply:
Comment is ok to be more verbose 

`Publish Release Notes to GitHub release Page`

Reply:
Done.